waffle_utils/file/search.py

```diff
@@ -3,7 +3,7 @@
 
 from natsort import natsorted
 
-from waffle_utils.image import SUPPORTED_IMAGE_EXTENSION
+from waffle_utils.image import SUPPORTED_IMAGE_EXTENSIONS
 from waffle_utils.video import SUPPORTED_VIDEO_EXTENSION
```

Comment:
Should it be changed to `SUPPORTED_VIDEO_EXTENSIONS` too?

Reply:
lol. I didn't notice that. That's from copilot. I'll apply that to `SUPPORTED_VIDEO_EXTENSION`, too.